.env

```
@@ -0,0 +1,22 @@
+
+# Server
+HOST=0.0.0.0
+PORT=1337
+
+# Secrets
+APP_KEYS=Ha0IPsjnA6UsaH0v9AxaGQ==,WRf3bf+NLgdmFTs9MS2LLg==,pMUJb9fRzkalIFfLGB/J9A==,JGGWwIEI6qi8og5SR5WkBA==
+API_TOKEN_SALT=kuRx0Z9UPSjWdXWEpRlnEw==
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The API_TOKEN_SALT key should go before the APP_KEYS key

(UnorderedKey)